test_utils/test_utils.py

```diff
@@ -297,19 +301,31 @@ def make_enum(
     name: str,
     package: str = 'foo.bar.v1',
     module: str = 'baz',
-    values: typing.Sequence[typing.Tuple[str, int]] = (),
+    values: typing.Sequence[typing.Tuple[str, int, bool]] = (),
     meta: metadata.Metadata = None,
     options: desc.EnumOptions = None,
+    is_deprecated: bool = False,
 ) -> wrappers.EnumType:
     enum_value_pbs = [
         desc.EnumValueDescriptorProto(name=i[0], number=i[1])
         for i in values
     ]
+
+    enum_index = 0
+    for enum_value in enum_value_pbs:
+        if len(values[enum_index]) > 2:
+            enum_value.options.deprecated = values[enum_index][2]
```

Comment:
This looks quite subtle until I realized that the data type `typing.Tuple[str, int, bool]` stands for `(enum_name, enum_value, is_deprecated)` from reading the new test `test_enum_values_deprecated` below.  Maybe comment about this test util code a little bit here to make the code easier to understand locally.

Reply:
Fixed in ee45a3d